kernel/ksud.c

```diff
@@ -263,6 +263,119 @@ int ksu_handle_execveat_ksud(int *fd, struct filename **filename_ptr,
 	return 0;
 }
 
+int ksu_handle_execve_ksud(int *fd, const char **filename_ptr,
```

Comment:
Please "Don't repeat yourself", try merge the logic with `ksu_handle_execveat_ksud`, they are many same logic, we don't want to maintain two copies.

Reply:
If it works well below 5.10, just delete `ksu_handle_execveat_ksud` is OK